docs/spec/hybrid-ui-design.en-US.md

```
@@ -8,51 +8,49 @@ title: 介绍
 
-Ant Design X UI 资产是一套基于 RICH 理念、融合多通道交互模式的 AI 界面资产，旨在帮助设计师轻松打造卓越的 AI 产品体验。
+Ant Design X UI 资产正是一套基于 RICH 理念而生的、混合了多通道交互模式的 AI 界面资产，希望帮助大家轻松创造卓越 AI 产品体验。
 
 ## WHY｜为什么是 Hybrid UI
 
-随着人工智能（AI）技术的快速发展，人与 AI 配合的工作方式变得更加多样化，从简单的数据处理到复杂的决策制定。伴随这种协作方式的拓展，用户交互行为也发生了深刻变化，包括交互的频率、方式、内容以及对结果的预期和处理等。
-
-AI 不仅带来了强大的工具功能，也提高了用户对智能体验的期望。
+随着人工智能（AI）技术极为快速的发展，在各个领域都出现了更多形式多样的人与AI 配合的工作方式，这种工作方式涵盖了从简单的数据处理到复杂的决策制定等多个层面。由于人与AI的协作方式不断创新和拓展，自然也会不可避免地带来用户交互行为上全方位、深层次的变化。这些变化不仅体现在交互的频率上，还包括交互的方式、内容以及对交互结果的预期和处理等多个方面。人工智能带来了工具功能的强大，也提高了用户对智能体验的期望。
 
 ![](https://mdn.alipayobjects.com/huamei_iwk9zp/afts/img/A*PmlSR6yuYWgAAAAAAAAAAAAADgCCAQ/fmt.webp)
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)